unit-tests/run_test_tokenizer.sh

```diff
@@ -18,11 +18,11 @@ function exec_test()
 	# local DIFF_RES=$(diff <(echo "$2") <(echo "$RES"))
 	# cat -e <(echo $DIFF_RES)
 	if [[ $RES == $3 ]]; then
-		echo "${BLUE}$1 ${BOLD}${GREEN}OK${RESET}"
+		printf "${BLUE}%2s ${BOLD}${GREEN}OK${RESET}\n" "$1"
```

Comment:
Better `printf` than `echo` 👍

Reply:
ok so it's about potability? will look into it.